demo/data/flags.json

```diff
@@ -1,26 +1,45 @@
-[
-    {
+{
+    "beta": [{
```

Comment:
What was the thinking behind this? Are we creating a lot of work for ourselves maintaining two collections of flags? And there's only one code base right so the feature flag needs to either exist in the code base or not, regardless of which environment we're running on.